src/asynk/async_queue.rs

```diff
@@ -185,38 +173,18 @@ where
 pub struct AsyncQueueTest<'a> {
     #[builder(setter(into))]
     pub transaction: Transaction<'a>,
-    #[builder(default = false, setter(into))]
-    pub duplicated_tasks: bool,
 }
 
 #[cfg(test)]
-impl<'a> AsyncQueueTest<'a> {
-    pub async fn find_task_by_id(&mut self, id: Uuid) -> Result<Task, AsyncQueueError> {
-        let row: Row = self
-            .transaction
-            .query_one(FIND_TASK_BY_ID_QUERY, &[&id])
-            .await?;
+#[async_trait]
+impl AsyncQueueable for AsyncQueueTest<'_> {
+    async fn find_task_by_id(&mut self, id: Uuid) -> Result<Task, AsyncQueueError> {
+        let transaction = &mut self.transaction;
 
-        let task = AsyncQueue::<NoTls>::row_to_task(row);
+        let task = AsyncQueue::<NoTls>::find_task_by_id_query(transaction, id).await?;
```

Comment:
these two lines can be replaced with 
```rust
AsyncQueue::<NoTls>::find_task_by_id_query(transaction, id).await
```

Reply:
[Comments adressed](https://github.com/ayrat555/fang/pull/71/commits/9c6c34faf584651f259dec069083509f78ce5934)